code/main/media/mediaModule.brs

```diff
@@ -51,7 +51,7 @@ function _adb_MediaModule(configurationModule as object, edgeModule as object) a
             eventType = eventData.xdmData.xdm.eventType
             clientSessionId = eventData.clientSessionId
 
-            if eventType <> m._CONSTANTS.MEDIA.EVENT_TYPE.SESSION_START and clientSessionId <> m._sessionManager.getActiveClientSessionId() then
+            if not _adb_stringEqualsIgnoreCase(eventType, m._CONSTANTS.MEDIA.EVENT_TYPE.SESSION_START) and clientSessionId <> m._sessionManager.getActiveClientSessionId() then
```

Comment:
There should be an `or` separating these two checks.

Reply:
It is supposed to be and. Added comments to clarify the logic.